### ISSUE TITLE
Add support for Smartenit EZIOXX

### ISSUE THREAD

**Issue by danielduerr** [pyinsteon/pyinsteon#209]
Hi,

Would you be open to adding support for the [Smartenit EZIOXX](https://smartenit.com/product-category/input-output-controllers/) input/output controller devices?  I have 3 EZIO2X4 devices which I use for monitoring dry contact inputs, analog temperature sensors, and a couple output relays as well.  The basic device support should be pretty straightforward/quick to add.  The analog inputs are a little more complex, but pretty powerful in certain applications.

I'm not sure exactly how the analog inputs would be made useful in Home Assistant, but I suspect it would involve passing the raw value to a custom helper script that could then transform the value into something useful.  For example, in my case I have an analog temperature sensor which requires some voltage math to convert into degrees F.

I have a functional PHP driver implemented from my defunct automation controller project which I would be happy to share with you if that is helpful.  But the [command set documentation](http://docs.smartenit.com/command_sets/ezioxx_command-set.pdf) is pretty straightforward and largely follows typical Insteon patterns.

Thank you for the consideration!

**Comment by teharris1** [comment]
Do you have your PHP code in a repository somewhere that I can look at? It may be helpful to capture the nuances of these devices. I am hoping to get this into the 2022.8.x release. I only have a week and a half to get that done, otherwise it moves into a later release.

**Comment by danielduerr** [comment]
@teharris1 absolutely!  I had emailed you privately a month or so ago from my personal gmail address, planned to offer you a clone of my git repo so you could reference my code.  Can we connect up that way, or do you need me to share the file here?  I think it'll make the most sense for you to get the entire repo vs. just the one file.  I'm happy to connect up and walk you through some code too.  I'm sure you can get this implemented pretty quickly.  Let me know.

**Comment by teharris1** [comment]
Yeah, that would be a good idea. I have not touched PHP code in about 20 years so it would be a lot faster if you walked me through it. Are you on Discord?

**Comment by danielduerr** [comment]
I am not, but could join you there if you prefer.  I mainly use Slack and Google Meet.  I emailed you at your commiter email I found in your pyinsteon code.  Maybe we can organize there?

**Comment by teharris1** [comment]
That sounds good. I remember you emailing me at my committer account but cannot find that email to reply to. Can you send me a reply and we can figure out how/when to connect?

**Comment by danielduerr** [comment]
Absolutely.  I just resent the email to you.
